README.md

```diff
@@ -9,28 +10,62 @@ It tests clean installation and upgrade for integration packages in CentOS, Suse
 ### `/linux`
 
 Usage and defaults:
+
+Test local packages produced in the `dist` folder:
 ```yaml
-    - name: Test packages installation
-      uses: newrelic/integrations-pkg-test-action/linux@v1
-      with:
-        tag: ${{ env.TAG }} # Required, trailing v is stripped automatically if found
-        integration: 'nri-${{ env.INTEGRATION }}' # Required, with nri- prefix
+- name: Test packages installation
+  uses: newrelic/integrations-pkg-test-action/linux@v1
+  with:
+    tag: ${{ env.TAG }} # Required, trailing v is stripped automatically if found
+    integration: 'nri-${{ env.INTEGRATION }}' # Required, with nri- prefix
+```
+
+Test packages uploaded to the staging repos:
+```yaml
+- name: Test staging repo
+  uses: newrelic/integrations-pkg-test-action/linux@v1
+  with:
+    tag: ${{ env.TAG }} # Does *NOT* specify which version is downloaded from the remote repo, `-show_version` output *WILL* be checked against this.  
```

Comment:
I would add a small comment here to clarify that we always test the latest version available in the repo.

Reply:
Comment is a bit unclear, I'll update it. However with the latest refactor is now easier to implement such a thing, if we want to :)